src/CommunityToolkit.Maui.MediaElement/Primitives/WindowsEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+namespace CommunityToolkit.Maui.Primitives;
+/// <summary>
+/// 
+/// </summary>
+public class WindowsEventArgs : EventArgs
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	public object? data { get; }
```

Comment:
not sure if you will change this latter, but can we make it strongly Typed?

Reply:
I will make sure it is strongly typed or change it to something else.